helpers/aws_validation.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+import datetime
+import hashlib
+import hmac
+
+from urllib.error import HTTPError
+from urllib.request import Request, urlopen
+
+
+class AwsValidation:
```

Comment:
I wasn't sure what convention you like to follow with that... I couldn't find other examples in the codebase 🤦 what do you prefer?

Reply:
I prefer `AWSValidation` because it is Amazon Web Services even this point does not make sense in `snake_case`. 
We won't write `a_w_s_validation`!
Having that said, that's my preference, not our convention, so do whatever you want.